### ISSUE TITLE
Showing correct answers in Rapid Fire game after completion.

### ISSUE THREAD

**Issue by shreyans29** [systers/malaria-app-android#243]
While playing the rapid fire game, when the answer selected is wrong, wouldn't it be better if we could show what the correct answer is ? That would help the user to learn as well.

**Comment by anubhakushwaha** [comment]
@shreyans29 As of now the wrong answer is indicated by a grey coloration,yes I agree that users should know the right answer.I guess a better way would be to display the correct answer incase he/she answers it wrong after the timer ends and before proceeding to the next question.Willing to work?

**Comment by shreyans29** [comment]
yes, sure. Is it okay if there is a delay, I have my final exams going on.
